packages/storybook/src/stories/index.js

```diff
@@ -1,5 +1,11 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
 import 'normalize.css';
 import './styles.css';
+import readme from '../../../../README.md';
+
+// eslint-disable-next-line react/no-danger
+storiesOf(' Welcome', module).add('to govuk-frederic', () => <div style={{ padding: '10px' }} className="markdown-body" dangerouslySetInnerHTML={{ __html: readme }} />);
```

Comment:
Real shame to use dangerouslySetInnerHTML - could you add a comment to the code as to why this is required plx?

Reply:
I will check this out if I can today to check toooooo

Reply:
I'm just not sure of another way of doing it. 🤔 If we just output the readme, it's not in markdown but it is rendered as an HTML string. Yep, if you can see a better way then great.

Reply:
Sorted! https://github.com/UKHomeOffice/govuk-react/pull/390/files